core/domain/translatable_object_registry.py

```diff
@@ -19,16 +19,17 @@
 import inspect
 
 from extensions.objects.models import objects
+from typing import Any, Dict, List
 
 
 class Registry:
     """Registry of all translatable objects."""
 
     # Dict mapping object class names to their classes.
-    _translatable_objects_dict = {}
+    _translatable_objects_dict: Dict[str, Any] = {}
```

Comment:
Why `Dict[str, Any]`? (especially the `Any` part)

Reply:
sorry i think i pushed the wrong commits . Will mend it asap !